### ISSUE TITLE
Please provide an option to preview the pages while creating it

### ISSUE THREAD

**Issue by atapas** [Hashnode/user-feedback#186]
**Is your feature request related to a problem? Please describe.**
While creating a page I can not preview the page before creating. It would be great to have an option to preview the pages as we build just like the one we have for story posting.

![image](https://user-images.githubusercontent.com/3633137/86199608-2d60a700-bb78-11ea-8a07-05dc43088d0c.png)

**Describe the solution you'd like**
Provide a Preview option. Clicking the preview option we can see how the page content is getting built.

**Describe alternatives you've considered**
No alternatives. Just blind creation

**Additional context**
Lacking of this feature stopping me from creating the pages as I am not confident to create one without previewing it.

**Comment by fazlerocks** [comment]
Yes, this will be upgraded with the new blog dashboard we are working on. @atapas
